src/store/model-store.js

```diff
@@ -12,7 +12,8 @@ let models = [
         isFaulty: false,
         faultDescription: '',
         price: 10.50,
-        count: 20
+        count: 20,
```

Comment:
Why did you add these properties to these items? It's not even all the applicable properties.